kubeadm/buildx.psm1

```diff
@@ -0,0 +1,58 @@
+function Set-Builder()
+{
+    $env:DOCKER_CLI_EXPERIMENTAL = "enabled"
+    & docker buildx create --name img-builder --use
+}
+
+function New-Build()
+{
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$name,
+        [Parameter(Mandatory = $true)]
+        [ValidateSet("docker", "registry")]
+        [string]$output,
+        [Parameter(Mandatory = $true)]
+        [string[]]$args
+    )
+
+    $command = "docker buildx build --platform windows/amd64 --output=type=$output -f Dockerfile -t $name"
+    foreach($arg in $args)
+    {
+        $command = "$command --build-arg=$arg"
+    }
+    $command = "$command ."
+    Invoke-Expression $command
+}
+
+function Add-Manifest([string]$name, [string[]]$items, [string[]]$bases)
```

Comment:
This looks to push the manifest as well.  Could you rename it "Push-Manifest" to reflect that?

Reply:
Ok. I did it.